server/views/components/opening-hours/index.njk

```diff
@@ -0,0 +1,28 @@
+<div class="opening-hours js-opening-hours js-tabs {% for modifier in modifiers %}opening-hours--{{ modifier }} %} {% endfor %}">
+  <ul class="opening-hours__tablist js-tablist">
+    {% for place in places %}
+      <li class="opening-hours__tabitem js-tabitem">
+        <a class="opening-hours__tablink js-tablink" href="#{{ place.id }}">{{ place.name }}</a>
+      </li>
+    {% endfor %}
+  </ul>
+  {% for place in places %}
+    <table id="panel-{{ place.id }}"class="opening-hours__table js-tabpanel">
+      <caption id="{{ place.id }}" class="opening-hours__caption js-tabfocus">{{ place.name }}</caption>
+      <thead class="opening-hours__thead">
+        <tr>
+          <th scope="col">Day</th>
+          <th scope="col">Times</th>
+        </tr>
+      </thead>
+      <tbody>
+        {% for day in place.days %}
+          <tr class="opening-hours__tr">
+            <td class="opening-hours__td">{{ day.name }}</td>
+            <td class="opening-hours__td">{{ day.hours }}</td>
+          </tr>
+        {% endfor %}
+      </tbody>
+    </table>
+  {% endfor %}
+</div>
```

Comment:
We could use [this](https://schema.org/openingHours)?
Although we could probably stick it in the JSON LD of the site.